notebooks/fixtures/config_test_file_c.nml

```diff
@@ -0,0 +1,4 @@
+&values
+  greeting = "Hello"
+  recipient = "World"
+/
```

Comment:
```suggestion
/
```

Reply:
@lexbuko22 It looks like you are resolving these comments without actually applying the suggestions. I'm unresolving them. Please ping us in Slack if you don't seem to be able to apply the suggestions via the web interface.

Reply:
Or maybe I haven't entered the suggestions correctly. I think you need to click _Commit suggestion_, or _Add suggestion to batch_ (and then later commit the batch, though I haven't tried this myself) instead of _Resolve conversation_. Apologies if that's what you've already done -- please let me know if so and I'll see if there's something wrong with the way I'm making commit-able suggestions.

Reply:
This was my mistake, I did the add suggestion to batch for each of these, but it looks like it only captured the ending "/" and not the file updates. I will get this corrected and recommit.

Reply:
All files should end with a newline, and the GitHub diff shows that some do not:
<img width="133" alt="image" src="https://github.com/user-attachments/assets/bdf21bb6-0f3e-4723-983c-e406581dd45b">
The newline is invisible in the diff, but I'd hope that committing the proposed suggestion will add the newline. Please try and let us know if it doesn't! (And if it doesn't you can add those newlines manually and push.)